code/game/machinery/computer/camera_advanced.dm

```diff
@@ -76,7 +76,7 @@
 	playsound(src, 'sound/machines/terminal_off.ogg', 25, FALSE)
 
 /obj/machinery/computer/camera_advanced/check_eye(mob/user)
```

Comment:
As it stands, the advanced camera console shows 1 tile of view, while the regular camera console gives actual vision.